UICatalog/Scenarios/TabViewExample.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Terminal.Gui;
+using static UICatalog.Scenario;
+
+namespace UICatalog.Scenarios {
+
+	[ScenarioMetadata (Name: "Tab View", Description: "Demos TabView control with limited screen space in Absolute layout")]
+	[ScenarioCategory ("Controls")]
+	class TabViewExample : Scenario {
+
+		TabView tabView;
+
+		MenuItem miShowTopLine;
+		MenuItem miShowBorder;
+		MenuItem miTabsOnBottom;
+
+		public override void Setup ()
+		{
+			Win.Title = this.GetName ();
+			Win.Y = 1; // menu
+			Win.Height = Dim.Fill (1); // status bar
+			Top.LayoutSubviews ();
+
+			var menu = new MenuBar (new MenuBarItem [] {
+				new MenuBarItem ("_File", new MenuItem [] {
+
+					new MenuItem ("_Add Blank Tab", "", () => AddBlankTab()),
+
+					new MenuItem ("_Clear SelectedTab", "", () => tabView.SelectedTab=null),
+					new MenuItem ("_Quit", "", () => Quit()),
+				}),
+				new MenuBarItem ("_View", new MenuItem [] {
+					miShowTopLine = new MenuItem ("_Show Top Line", "", () => ShowTopLine()){
```

Comment:
Note how you use different words to describe this flag in UI ("Show Top Line") vs. code (`ShowHeaderOverline). Please consider how this would all be simpler if you used consistent terminology. I've always found that "what makes sense to the end-user will make sense to the programmer" (not the other way around).